stl/inc/tuple

```diff
@@ -163,20 +163,21 @@ struct _Tuple_val { // stores each value in a tuple
     constexpr _Tuple_val(_Other&& _Arg) : _Val(_STD forward<_Other>(_Arg)) {}
 
     template <class _Alloc, class... _Other, enable_if_t<!uses_allocator_v<_Ty, _Alloc>, int> = 0>
-    _Tuple_val(const _Alloc&, allocator_arg_t, _Other&&... _Arg) : _Val(_STD forward<_Other>(_Arg)...) {}
+    _CONSTEXPR20 _Tuple_val(const _Alloc&, allocator_arg_t, _Other&&... _Arg) : _Val(_STD forward<_Other>(_Arg)...) {}

```

Comment:
`_Tuple_val` is internal machinery; we can simply make these `constexpr`. (Here, 172, and 179.)

Reply:
Done